fastmcp-auth/test_fastmcp_auth.py

```
@@ -0,0 +1,372 @@
+
+This script tests all the authentication functionality including:
+- JWT token generation with email as subject (gyliu513@gmail.com)
+- Token validation using FastMCP BearerAuthProvider
+- Access token claims extraction
+- Google public key integration
+"""
+
+import sys
+import os
+import time
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import

(F401)